Parser/Client/Browser.php

```diff
@@ -231,6 +231,7 @@ class Browser extends AbstractClientParser
         'OY' => 'Origyn Web Browser',
         'OV' => 'Openwave Mobile Browser',
         'OW' => 'OmniWeb',
+        'YX' => 'Onyx',
```

Comment:
https://en.wikipedia.org/wiki/Oregano_(web_browser)
it would probably be more correct to leave the name `Oregano Browser`

Reply:
Then, I think is much better to leave it as `Oregano`, and improve that detection. Because is the same browser.

Reply:
you are right, it is better to make one name `Oregano`